privacyidea/static/components/token/views/token.import.html

```diff
@@ -82,11 +82,12 @@ <h3 translate>Import tokens from a file</h3>
             </p>
             <select class="form-control"
                     ng-model="form.validateMAC">
-                <option ng-repeat="(key, desc) in {no_check: 'Do not verify the authenticity' | translate,
-                  check_fail_soft : 'Skip tokens that can not be verified' | translate,
-                  check_fail_hard : 'Abort operation on unverifiable token' | translate}"
+                <option ng-repeat="(key, desc) in {
+                    no_check: 'Do not verify the authenticity',
+                    check_fail_soft : 'Skip tokens that can not be verified',
+                    check_fail_hard : 'Abort operation on unverifiable token'}"
                         value="{{ key }}">
-                    {{ desc }}
+                    {{ desc | translate}}
```

Comment:
I think this works out, if the strings 'Do not verify the authenticity' are already translated.

But if we change this string, grunt will not identify the string as translateable and we will not be able to translate changes in the future.